tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+import wazuh_testing.fim as fim
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+
+
+# Helper functions
+
+def extra_configuration_after_yield():
+    fim.delete_registry(fim.registry_parser[key], sub_key_1, fim.KEY_WOW64_32KEY)
+
+
+def check_event_type_and_path(fim_event, monitorized_registry):
+    check_event = False
+
+    if fim_event['type'] == 'added':
+        registry_event_path = fim_event['path']
+        if monitorized_registry.lower() == registry_event_path.lower():
+            check_event = True
+
+    return check_event
+
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+
+# Variables
+
+key = "HKEY_LOCAL_MACHINE"
+classes_subkey = os.path.join('SOFTWARE', 'Classes')
+
+sub_key_1 = os.path.join(classes_subkey, 'testkey')
+sub_key_2 = os.path.join(classes_subkey, 'Testkey')
+
+registry_1, registry_2 = os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+
+monitoring_modes = ['scheduled']
+
+# Configurations
+
+conf_params = {'WINDOWS_DUPLICATED_REGISTRY_1': registry_1,
+               'WINDOWS_DUPLICATED_REGISTRY_2': registry_2}
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_duplicated_registry.yaml')
+parameters, metadata = fim.generate_params(extra_params=conf_params, modes=monitoring_modes)
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+registry_list = [(key, sub_key_1, fim.KEY_WOW64_32KEY), (key, sub_key_2, fim.KEY_WOW64_32KEY)]
+
+
+# Fixtures
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# Test
+
+@pytest.mark.parametrize('key, subkey1, subkey2, arch', [(key, sub_key_1, sub_key_2, fim.KEY_WOW64_32KEY)])
+def test_registry_duplicated_entry(key, subkey1, subkey2, arch, get_configuration, configure_environment,
+                                   restart_syscheckd, wait_for_fim_start):
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over.  That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
The reasons for this change are:
- Faster testing
- More encapsulated tests

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/7bc4f0c313eb06fcbe3a8fed8f4a70138048e379)